### ISSUE TITLE
Ship-building catbombs after getting a ship

### ISSUE THREAD

**Issue by SunDwarf** [KanColleTool/kcsrv#33]
No idea why, dock data _seems_ correct.  

This doesn't affect the data itself, it's purely a client-side issue.

**Comment by Xaiwaker** [comment]
Yeah, after choosing Fubuki and going through all the text, it catbombs consistently.

**Comment by SunDwarf** [comment]
This unfortunately I don't know how to fix, because it's completely non-determined as to why.
